content/_data/authors/strangelookingnerd.adoc

```diff
@@ -0,0 +1,7 @@
+---
+name: "Daniel Krämer"
+github: strangelookingnerd
+linkedin: daniel-krämer
```

Comment:
```suggestion
name: "Daniel Krämer"
github: strangelookingnerd
linkedin: daniel-kr%C3%A4mer
```

Tested locally:

> <img width="1260" height="606" alt="Image" src="https://github.com/user-attachments/assets/cdc63133-a0a7-4cc4-a1d9-de15ea60d86b" />

Reply:
I checked the current deployment, it renders my name correctly and linkedin is also working

Reply:
Sorry I didn't noticed you had already pushed a fix before posting this suggestion 😅